jboss-image-streams.json

```diff
@@ -959,10 +973,9 @@
                         "annotations": {
                             "description": "JBoss Data Grid 6.5 S2I images.",
                             "iconClass": "icon-jboss",
-                            "tags": "datagrid,jboss,xpaas",
-                            "supports":"datagrid:6.5,xpaas:1.4",
-                            "version": "1.4",
-                            "openshift.io/display-name": "Red Hat JBoss Data Grid 6.5"
```

Comment:
Did we intend to remove the display-name?

Reply:
no idea why it was removed, adding back